docs/source/defaults.rst

```diff
@@ -0,0 +1,47 @@
+.. toggle::
+
+    .. testcode:: default, 1, 2, 3, 4, 5
```

Comment:
I think this can be just 
```suggestion
    .. code-block:: python
```

Reply:
If I write `.. code-block:: python` the block would not run and hence the default setup would no be defined. This is also confirmed when I try it.